source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -219,6 +219,10 @@ For any continuous synchronization use cases with ``mongosync``, ensure that
 ``mongosync`` commits before cutting over from the source to the 
```

Comment:
this line starting with "For any continuous synchronization" is now redundant with the new textbox so we can remove it.